segmentation/include/pcl/segmentation/grabcut_segmentation.h

```diff
@@ -399,9 +394,9 @@ namespace pcl
       // Storage for N-link weights, each pixel stores links to nb_neighbours
       struct NLinks
       {
-        NLinks () : nb_links (0), indices (0), dists (0), weights (0) {}
+        NLinks () :  indices (0), dists (0), weights (0) {}
```

Comment:
indices (0) are Indices and dists (0), weights (0) are all std::vectors - which don't need this initialization?